README.rst

```diff
@@ -97,6 +97,10 @@ This field takes three optional arguments:
 - ``auto_bbox``: If ``True``, the GeoJSON object will include
   a `bounding box <https://datatracker.ietf.org/doc/html/rfc7946#section-5>`_,
   which is the smallest possible rectangle enclosing the geometry.
+- ``transform`` (defaults to ``4326``): If ``None`` (or the input geometry does not have
+  a SRID), the GeoJSON's coordinates will not be transformed. If any other `spatial
```

Comment:
"If any other" this line is not clear, can you please rephrase it?

Reply:
I have rephrased these docs after setting the default value to `None`. I hope it is easier to understand now. What do you think?

Reply:
ping  @auvipy